### DISCUSSION TITLE
Advanced log search not working

### DISCUSSION THREAD

**Discussion by DM2PF** [magicbug/Cloudlog#2994] [Q&A]
Hi,

not sure if that's a bug or some fault on my side - the advanced log search is not working after the last update I did (not sure which exact version I updated from - might be 2-3 months), see screenshot.

Did anyone face the same error and can give me hints how to resolve it?

Thanks!
Paul

<img width="1438" alt="Screenshot 2024-02-26 at 21 37 12" src="https://github.com/magicbug/Cloudlog/assets/6350381/1562f403-bc3d-4605-a41f-fdeb0d3de54d">

**Comment by magicbug** [comment]
Does appear to be an issue with language files I'll take a look later

**Comment by magicbug** [comment]
This should now be fixed in the dev branch

**Reply by DM2PF** [reply]
It's working now - thanks a lot! :)
